src/OpenColorIO/transforms/builtins/ACES.cpp

```diff
@@ -598,26 +843,190 @@ void RegisterAll(BuiltinTransformRegistryImpl & registry) noexcept
+        registry.addBuiltin("ACES-OUTPUT--ACES_to_CIE-XYZ_SDR-VIDEO-D60sim-D65_1.0", 
                             "Component of ACES Output Transforms for SDR D65 video simulating D60 white",
                             ACES2065_1_to_CIE_XYZ_video_d60sim_1_0_Functor);
     }
 
     {
-        auto ACES2065_1_to_CIE_XYZ_hdr_video_1000nits_1_1_Functor = [](OpRcPtrVec & ops)
+        auto ACES2065_1_to_CIE_XYZ_cinema_d60sim_dci_1_0_Functor = [](OpRcPtrVec & ops)
+        {
+            ACES_OUTPUT::Generate_RRT_preamble_ops(ops);
+
+            ACES_OUTPUT::Generate_tonecurve_ops(ops);
+
+            ACES_OUTPUT::Generate_roll_white_d60_ops(ops);
+
+            CreateRangeOp(ops, 
+                          RangeOpData::EmptyValue(), 0.918,  // don't clamp low end
+                          RangeOpData::EmptyValue(), 0.918,  // don't clamp low end
+                          TRANSFORM_DIR_FORWARD);
+
+            static constexpr double scale     = 0.96;
+            static constexpr double scale4[4] = { scale, scale, scale, 1. };
+            CreateScaleOp(ops, scale4, TRANSFORM_DIR_FORWARD);
+
+            MatrixOpData::MatrixArrayPtr matrix
+                = rgb2xyz_from_xy(ACES_AP1::primaries);
+            CreateMatrixOp(ops, matrix, TRANSFORM_DIR_FORWARD);
+
+            MatrixOpData::MatrixArrayPtr matrix2
+                = build_vonkries_adapt(WHITEPOINT::DCI_XYZ, WHITEPOINT::D65_XYZ, ADAPTATION_BRADFORD);
+            CreateMatrixOp(ops, matrix2, TRANSFORM_DIR_FORWARD);
+        };
+
+        registry.addBuiltin("ACES-OUTPUT--ACES_to_CIE-XYZ_SDR-CINEMA-D60sim-DCI_1.0", 
+                            "Component of ACES Output Transforms for SDR DCI cinema simulating D60 white",
+                            ACES2065_1_to_CIE_XYZ_cinema_d60sim_dci_1_0_Functor);
+    }
+
+    {
+        auto ACES2065_1_to_CIE_XYZ_cinema_d65sim_dci_1_1_Functor = [](OpRcPtrVec & ops)
+        {
+            ACES_OUTPUT::Generate_RRT_preamble_ops(ops);
+
+            ACES_OUTPUT::Generate_tonecurve_ops(ops);
+
+            ACES_OUTPUT::Generate_roll_white_d65_ops(ops);
+
+            CreateRangeOp(ops, 
+                          RangeOpData::EmptyValue(), 0.908,  // don't clamp low end
+                          RangeOpData::EmptyValue(), 0.908,  // don't clamp low end
+                          TRANSFORM_DIR_FORWARD);
+
+            static constexpr double scale     = 0.9575;
+            static constexpr double scale4[4] = { scale, scale, scale, 1. };
+            CreateScaleOp(ops, scale4, TRANSFORM_DIR_FORWARD);
+
+            AP1_to_CIE_XYZ_D65::GenerateOps(ops);
+
+            MatrixOpData::MatrixArrayPtr matrix2
+                = build_vonkries_adapt(WHITEPOINT::DCI_XYZ, WHITEPOINT::D65_XYZ, ADAPTATION_BRADFORD);
+            CreateMatrixOp(ops, matrix2, TRANSFORM_DIR_FORWARD);
+        };
+
+        registry.addBuiltin("ACES-OUTPUT--ACES_to_CIE-XYZ_SDR-CINEMA-D65sim-DCI_1.1", 
```

Comment:
As the builtin names end up to be public, I have two questions:

- Why did you change `ACES2065-1` to `ACES`?
I found it less readable when present in a config file.

- What is the meaning of `--`?   
If that's a way to add some `categories` (or `families`) to the builtin transforms a solution is to explicitly add them i.e. `registry.addBuiltin(tag, family tag, explanation, functor)` and use the family 'style' (i.e. `/`) in the config file.

Reply:
I had a similar thought about the `--`.